src/main/default/decisionMatrixDefinition/Risk_Score_Decision_Matrix.decisionMatrixDefinition-meta.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<DecisionMatrixDefinition xmlns="http://soap.sforce.com/2006/04/metadata">
+    <label>Risk Score Decision Matrix</label>
+    <processType>Bre</processType>
+    <type>Standard</type>
+    <versions>
+        <fullName>Risk_Score_Decision_Matrix_V1</fullName>
+        <columns>
+            <columnType>Input</columnType>
+            <dataType>Text</dataType>
+            <displaySequence>1</displaySequence>
+            <isWildcardColumn>false</isWildcardColumn>
+            <name>Scope</name>
+        </columns>
+        <columns>
+            <columnType>Input</columnType>
+            <dataType>Text</dataType>
+            <displaySequence>2</displaySequence>
+            <isWildcardColumn>false</isWildcardColumn>
+            <name>Severity</name>
+        </columns>
+        <columns>
+            <columnType>Output</columnType>
+            <dataType>Number</dataType>
+            <displaySequence>3</displaySequence>
+            <isWildcardColumn>false</isWildcardColumn>
+            <name>RiskScore</name>
+        </columns>
+        <decisionMatrixDefinition>Risk_Score_Decision_Matrix</decisionMatrixDefinition>
+        <label>Risk Score Decision Matrix V1</label>
+        <startDate>2024-03-06T11:07:13.000Z</startDate>
+        <status>Draft</status>
```

Comment:
Please clarify if this Decision Matrix must be deployed and kept as **inactive**, and if yes - then why?

Reply:
Status now changed to Active.